draco/analysis/beamform.py

```diff
@@ -607,6 +608,11 @@ def _process_catalog(self, catalog):
         if "position" not in catalog:
             raise ValueError("Input is missing a position table.")
 
+        if not hasattr(self, "epoch"):
+            raise PipelineRuntimeError(
+                "Epoch not set. Was the requested data not available?"
+            )
```

Comment:
https://github.com/radiocosmology/draco/issues/133

Reply:
We are gonna swap this `raise` for a `warning` log.

Reply:
We can remove the beamform.py changes @anjakefala